### DISCUSSION TITLE
Xdrip duplicate CGM data shown in AndroidAPS 3.0

### DISCUSSION THREAD

**Discussion by g7kbr67** [NightscoutFoundation/xDrip#1947] [General]
Hi all

I noticed a few days ago, Xdrip was duplicating CGM data.  Wonder can anyone confirm if this has been reported? I am using Xdrip nightly update on the 8th January. See attached photos.  

Android version 11, Using Libre 2, OOPS2, AndroidAPS 3.0 RC1

Regards

Paul
![image](https://user-images.githubusercontent.com/17815903/149993301-e1f23073-dc24-4aba-9dcb-ff19786c8b4d.jpeg) ![image](https://user-images.githubusercontent.com/17815903/149993257-8a422168-fab8-4318-8fb5-21c894516cc5.jpeg)

**Comment by tzachi-dar** [comment]
This is the first time that this is being reported for the libre2 (this was already reported and fixed for the g6)
Can you please try this with the latest version of xDrip(https://github.com/NightscoutFoundation/xDrip/releases/download/2022.01.15/xDrip-plus-20220115-c25c5dc.apk)

Let us know if this solves the issue for you.

**Comment by savek-cc** [comment]
@g7kbr67 Maybe you could export your xDrip-Database (3-dot menu at the top right, "import / export features" -> "Export Database" and have someone have a look what xDrip recorded in its own DB at that time? (Helped quite a lot to have that info in the Dexcom case)
